PSKoans/ExportedTypes.ps1

```diff
@@ -11,4 +25,33 @@ class KoanAttribute : System.Attribute {
     KoanAttribute() {
         $this.Position = [uint32]::MaxValue
     }
+
+    [void] InvokeHelpInfo() {
+        if ($this.HelpAction) {
+            $this.InvokeHelpAction()
+        }
+        elseif ($this.HelpPath) {
+            $this.InvokeHelpPath()
+        }
+        elseif ($this.HelpURL) {
+            $this.InvokeHelpURL()
+        }
+    }
+
+    [void] InvokeHelpAction() {
+        $this.HelpAction.InvokeReturnAsIs()
+    }
+
+    [void] InvokeHelpURL() {
+        Write-ConsoleLine "Confusion is only the beginning"
+        Write-ConsoleLine "Your defeat has only made you stronger"
+        Write-ConsoleLine ("Learn through Zen: {0}" -f $this.HelpURL)
+    }
+
+    [void] InvokeHelpPath() {
+        Write-ConsoleLine "Confusion is only the beginning"
+        Write-ConsoleLine "To become Master must be able to snatch the coin from my hand"
+        Write-ConsoleLine ("You are improving but you must learn more about {0}" -f $this.HelpPath)
```

Comment:
With how many help topics are going to be named `about_Something` I think the tail of this should read more like `... you must learn more from <topic>`